package.json

```diff
@@ -25,7 +25,8 @@
     "test": "jest --verbose",
     "test:ui": "babel-node test/main.js",
     "bundlesize": "bundlesize",
-    "lokalise:download": "lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID file download --format json --unzip-to ./src/locales/"
+    "lokalise:download": "lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID:$branch file download --format json --unzip-to ./src/locales/",
+    "lokalise:download-branch": "branch=$(git rev-parse --symbolic-full-name --abbrev-ref HEAD); lokalise2 --token $LOKALISE_TOKEN --project-id $LOKALISE_PROJECT_ID:$branch file download --format json --unzip-to ./src/locales/"
```

Comment:
Currently agreed with Charlotte that it will be when the feature is released since we do not have a "development" branch in. It's something I think we need to iron out at the meeting we'll be having this week about our design-dev process on Lokalise.